app/jsx/components/SortComp.jsx

```diff
@@ -41,10 +41,10 @@ class SortComp extends React.Component {
           <select name="rows" id="c-sort2" form="facetForm" value={ this.state.rows } onChange={ this.handleChange }>
           {/* ToDo: Make this more concise */}
           <option value="10">10</option>
-          {(this.props.count > 20) && <option value="20">20</option> }
-          {(this.props.count > 30) && <option value="30">30</option> }
-          {(this.props.count > 40) && <option value="40">40</option> }
-          {(this.props.count > 50) && <option value="50">50</option> }
+          <option value="20">20</option>
+          {(this.props.count > 20) && <option value="30">30</option> }
```

Comment:
i.e. if there are 24 pages, you should be able to select 30 rows so you can see them all